modules/launcher/src/main/java/org/mule/module/launcher/coreextension/DefaultMuleCoreExtensionManager.java

```diff
@@ -106,7 +106,7 @@ public void stop() throws MuleException
             {
                 extension.stop();
             }
-            catch (MuleException e)
+            catch (Exception e)
```

Comment:
If you want to be sure that core extensions are stopped, you must catch Throwable